actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -747,6 +747,20 @@ def date_field_tag(name, value = nil, options = {})
       # * <tt>:max</tt> - The maximum acceptable value.
       # * <tt>:step</tt> - The acceptable value granularity.
       # * <tt>:include_seconds</tt> - Include seconds and ms in the output timestamp format (true by default).
+      #
+      # ==== Examples
+      #
+      #   time_field_tag 'name'
+      #   # => <input id="name" name="name" type="time" />
+      #
+      #   time_field_tag 'time', '01:01'
+      #   # => <input id="time" name="time" type="time" value="01:01" />
+      #
+      #   time_field_tag 'time', nil, class: 'special_input'
+      #   # => <input class="special_input" id="time" name="time" type="time" />
+      #
+      #   time_field_tag 'time', '01:01', class: 'special_input', disabled: true
+      #   # => <input disabled="disabled" class="special_input" id="time" name="time" type="time" value="01:01" />
```

Comment:
Instead of showing options that are supported by other inputs, I think we should show `:min`, `:max`, `:step` and `:include_seconds`.